lib/ecto_neo4j.ex

```diff
@@ -54,9 +55,10 @@ defmodule Ecto.Neo4j do
   end
 
   def sort_column(col, query) do
-    columns = result_columns(query)
-    if hd(columns) do
-      columns |> Enum.map(fn {name, type} -> coerce(type, col[Atom.to_string(name)]) end)
+    cols = result_columns(query)
+    if hd(cols) do
+      coercer = fn({name, type}) -> coerce(type, col[Atom.to_string(name)]) end
```

Comment:
Interesting! This and the following line. I _think_ this is one of the things that's going to need a refactor, but it's cool for now, since none of the tests pass anyway. Defining that function inline seems a bit clunky to me, but I've no idea if that's normal idiomatic Elixir or not.